common/interfaces.go

```diff
@@ -0,0 +1,6 @@
+package common
+
+// Cloneable is an interface for objects that can be cloned
+type Cloneable interface {
```

Comment:
`Cloneable` seems too javaish, more golang style would be `Cloner` no? But I think this interface is too generic but used for a very specific purpose, Filters, I think we shove move it into the filter package `FilterCloner`. It seems out place for all filters to return a `common.Cloneable` since they are all filters. Or maybe we could have the method inside the `Filter` interface, and just return Filter?

Reply:
I agree with the nomenclature, it should be `Cloner`.

> But I think this interface is too generic but used for a very specific purpose, Filters, I think we shove move it into the filter package FilterCloner. It seems out place for all filters to return a common.Cloneable since they are all filters. Or maybe we could have the method inside the Filter interface, and just return Filter

It's not just Filters that implement this interface, `Policy` and `Policies` do so:

![image](https://github.com/aquasecurity/tracee/assets/3372117/ec96ee14-6332-445c-bc6a-c11cb93e524f)